### PR TITLE
Changed the build type to ament_python and Fixed package to run with ros2 run

### PR DESCRIPTION
**Pull request by ahcorde**
Fixed package to run with `ros2 run rqt_publisher rqt_publisher`. Related to this issue https://github.com/ros-visualization/rqt_console/issues/20

Signed-off-by: ahcorde <ahcorde@gmail.com>

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,11 +1,39 @@
...
+    maintainer_email='dthomas@osrfoundation.org',
+    keywords=['ROS'],
+    classifiers=[
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: BSD License',
+        'Programming Language :: Python',
+        'Topic :: Software Development',
+    ],
+    description=(
+        'RQT plugin for monitoring ROS processes.'
```
Comment (dirk-thomas) [inline setup.py:31]: Copy-n-paste mistake.

**Comment by dirk-thomas** [comment]
Please update the title that the PR actually changes the build type of the package. Same for all the other rqt-related PRs with similar changes.

**Comment by hidmic** [comment]
Rebased on top of `foxy-devel`. It can be backported to Dashing later on. I verified it can be `ros2 run` on Windows, and I see all icons.

Running CI up to `rqt_msg`, `rqt_srv`, `rqt_service_caller`, `rqt_top`, `rqt_shell`, `rqt_py_console`, `rqt_publisher`, `rqt_plot`:

* Linux [Build Status](http://ci.ros2.org/job/ci_linux/14396/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/9170/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/12071/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/14495/)

**Review by ivanpauno — APPROVED** [review]
LGTM!

Same comments than here https://github.com/ros-visualization/rqt_msg/pull/8#pullrequestreview-638063236.

**Pull request merged**